eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/highlight.filter.js

```diff
@@ -0,0 +1,15 @@
+﻿(function (angular) {
+    angular.module("equizModule").filter('highlight', HighlightFilter);

```

Comment:
Use the following convention:

angular
.module("equizModule")
.filter('highlight', HighlightFilter);